src/http/nodegen/errors/HttpException.ts

```diff
@@ -23,9 +23,7 @@ export class HttpException extends Error {
   set body(body: string | Record<string, any>) {
     this.rawBody = body;
     const fmt = HttpErrorsService.formatException(this);
-    if (fmt === this) {
-      this.rawBody = this.toJSON();
-    }
+    this.rawBody = fmt === this ? this.toJSON() : fmt;
```

Comment:
previously rawbody was body or json.. now it is json or fmt and the assignment of rawBody to body is redundant?

Reply:
So it has to assign twice - once before the call to format (which can call exception.body, which in this case is actually rawBody), and then once after to "update" the rawBody so subsequent calls return it.

The toJSON issue is to fix the issue where res.json results in circular json error (obj = { body: obj } )

But the main issue here is that if you return exception.body or { myCustomError: 'aw crap', original: exception.body } then it will show up in the res.json